x-pack/plugins/observability_solution/apm/public/components/shared/suggestions_select/index.tsx

```diff
@@ -7,7 +7,7 @@
 
 import { EuiComboBox, EuiComboBoxOptionOption } from '@elastic/eui';
 import { throttle } from 'lodash';
-import React, { useCallback, useState } from 'react';
+import React, { useCallback, useEffect, useRef, useState } from 'react';
```

Comment:
Q: Why do you need  `useEffect, useRef` here? I see that they are not used so I think we can remove them

```suggestion
import React, { useCallback, useState } from 'react';
```